format/tmux/formater.go

```diff
@@ -88,15 +98,20 @@ func (f *Formater) Format(w io.Writer, st *gitstatus.Status) error {
 	// overall working tree state
 	if f.st.IsInitial {
 		fmt.Fprintf(w, "%s%s [no commits yet]", f.Styles.Branch, f.st.LocalBranch)
-		goto fileCounts
+		f.flags()
+	} else {
```

Comment:
Instead of adding else, we could write the flags, write the state (WriteTo) and return.
Doing so you'd both have simplified your code and mine (removing `goto`) ! :)